test/model/content_test.dart

```diff
@@ -521,6 +521,16 @@ class ContentExample {
       blockUnimplemented('more text'),
     ]]),
   ]);
+
+  static const thematicBreaks =  ContentExample(
+    'parse thematic break (<hr>) in block context',
```

Comment:
nit: "in block context" is redundant because that's the only way `<hr>` can appear:
```suggestion
    'parse thematic break (<hr>)',
```

(there's a phrase like that for the test that produces `LineBreakNode`, but that's to contrast it with the "inside a paragraph" version that produces `LineBreakInlineNode`)